docs/Makefile

```diff
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    =
+SPHINXOPTS    = -W
```

Comment:
this raises error when there are warnings

Reply:
Good idea, should we also do this in Pyro and Funsor?